addons/l10n_sa/views/report_templates_views.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="l10n_sa_additional_footer">
+        <t t-set="use_additional_footer"
+        t-value="env['ir.config_parameter'].sudo().get_param('l10n_sa.l10n_sa_use_additional_footer') != 'False'"/>
+        <div t-if="report_type == 'pdf' and use_additional_footer and additional_footer_text" class="text-muted">
+            <span t-out="additional_footer_text"/>
+        </div>
+    </template>
+    <template id="l10n_sa_external_layout_striped" inherit_id="web.external_layout_striped">
+        <div class="text-center" position="inside">
+            <t t-call="l10n_sa.l10n_sa_additional_footer"/>
+        </div>
+    </template>
+    <template id="l10n_sa_external_layout_boxed" inherit_id="web.external_layout_boxed">
+        <div class="text-center" position="inside">
+            <t t-call="l10n_sa.l10n_sa_additional_footer"/>
+        </div>
+    </template>
+    <template id="l10n_sa_external_layout_bold" inherit_id="web.external_layout_bold">
+        <div t-attf-class="footer o_clean_footer o_company_#{company.id}_layout" position="inside">
+            <t t-call="l10n_sa.l10n_sa_additional_footer"/>
+        </div>
+        <xpath expr="//div[hasclass('col-4')]" position="inside">
+            <t t-call="l10n_sa.l10n_sa_additional_footer"/>
```

Comment:
It's weird to have it twice. Are we sure it's not causing issue? Or that it's useful?
Like wouldn't the second be enough?
And I would prefer that instead, as there are several col-4 div in there (I have not tested and I'm not sure it's the right div but you can test and select the right one).
```suggestion
       xpath expr="//span[@t-field='company.report_footer']" position="after">
            <t t-call="l10n_sa.l10n_sa_additional_footer"/>
```